test/e2e/mcd_test.go

```diff
@@ -355,8 +355,20 @@ func TestReconcileAfterBadMC(t *testing.T) {
 	cs := framework.NewClientSet("")
 	bumpPoolMaxUnavailableTo(t, cs, 3)
 
-	// create a bad MC w/o a filesystem field which is going to fail reconciling
-	mcadd := createMCToAddFile("add-a-file", "/etc/mytestconfs", "test", "")
+	// create a bad MC that contains a valid ignition config that is not reconcilable
+	mcadd := &mcfgv1.MachineConfig{
```

Comment:
you're missing metadata.name at least  and test is failing - why don't we just re-use `createMCToAddFile` which creates a valid ignition config and then we append the networkd bits as you're doing below?

Reply:
gah. thank you! done.